src/main/java/de/mirkosertic/flightrecorderstarter/configuration/FlightRecorderDynamicConfiguration.java

```diff
@@ -24,9 +24,16 @@
 @ConfigurationProperties(prefix = "flightrecorder")
 public class FlightRecorderDynamicConfiguration {
 
+    public enum CleanupType {
+        TTL,
+        COUNT
+    }
```

Comment:
[❓]
move the enum declaration to a separate class `RecordingCleanupType`? To which package? Please give feedback.